spec/models/campaign_gift_purchase_spec.rb

```diff
@@ -73,23 +73,23 @@
       'supporter' => supporter.id,
       'nonprofit' => nonprofit.id,
       'subtransaction' => nil,
-      'subtransaction_payments' => [],
+      'payments' => [],
       'transaction_assignments' => [
         cgp_builder_to_id
       ]
     }
   end
 
-  let(:cgp_builder_to_id) do 
+  let(:cgp_builder_to_id) do
     {
       'id' => match_houid('cgpur'),
       'object' => 'campaign_gift_purchase',
       'type' => 'trx_assignment'
     }
   end
-  
```

Comment:
Please undo all these formatting changes. They definitely should be done at some point but it makes it hard to review the PR.

Reply:
Solved!